src/main/scala/org/broadinstitute/dsde/vault/VaultServiceActor.scala

```diff
@@ -33,11 +34,13 @@ class VaultServiceActor extends HttpServiceActor with ActorLogging {
 
   // this actor runs all routes
   def receive = runRoute(
-    uBAMIngest.routes ~ uBAMDescribe.routes ~ uBAMRedirect.routes ~
-    analysisIngest.routes ~ analysisDescribe.routes ~ analysisUpdate.routes ~ analysisRedirect.routes ~
-    lookupService.routes ~
-    swaggerService.routes ~ swaggerUiService
-    )
+    swaggerService.routes ~ swaggerUiService ~
+      logOpenAMRequest(log) {
```

Comment:
I think we should use a different and dedicated LoggingAdapter here. That way, we can easily route audit logs to different appenders - we could have one file for audits and one for developer debug messages, and even a third for the combination of the two.